### DISCUSSION TITLE
End of life support for v11.1.0

### DISCUSSION THREAD

**Discussion by pneumowave-finlay-mcsporran** [HackedByChinese/ng2-idle#208] [Q&A]
Hi,

We are in the process of applying for FDA certification and need to include information about the end-of-life dates for packages. Could someone please inform me if there are any available details regarding the end of support for ng-idle version 11.1.0?

**Comment by grbsk** [comment]
Greetings, 

As an unfunded open source project, I offer no EOL commitments or warranties and assume no liabilities as described in the Apache-2.0 license this project is distributed under. The software components are provided as-is. In general, I do not unpublish previously published versions unless there is significant security or similar issue, and you are free to use old versions as long as github and npm are hosting them free of charge to me. You are also free to branch the project and support it yourself, under the conditions of the license. Finally, I do accept PRs and could conceivably patch an older version and release it for you, but I personally do not have the resources to maintain multiple versions of this project. With community help, we do try to make sure we have working versions of ng-idle that reflect the currently supported versions of Angular itself.

I don't know if that's exactly what you need, but I hope it helps.
